.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: cimg/rust:1.65.0
+      - image: cimg/rust:1.75.0
```

Comment:
The updated version of the `ahash` library is not compatible with Rust 1.65.0. Therefore, I've updated Rust as the `hashlink` library has no a lock file.